examples/application_epg/main.tf

```diff
@@ -13,35 +13,149 @@ provider "aci" {
   insecure = true
 }
 
-resource "aci_tenant" "dev_tenant" {
-  name = "demo_tenant"
+resource "aci_tenant" "terraform_tenant" {
+  name        = "static_leaf_tenant"
+  description = "This tenant is created by terraform"
 }
 
 resource "aci_application_profile" "test_ap" {
-  tenant_dn   = aci_tenant.dev_tenant.id
-  name        = "demo_ap"
-  annotation  = "tag"
+  tenant_dn   = aci_tenant.terraform_tenant.id
+  name        = "test"
   description = "from terraform"
```

Comment:
suggestion:
`description  =  "This application profile is created by terraform"`

Apply it to the rest of the objects when appropriate or when the description is missing. But this is just a suggestion for consistency in the example.